Automattic/woocommerce-payments
# Multi-Currency track enabled currencies update

Fixes #2760 

#### Changes proposed in this Pull Request
Track enabled currencies update using the `submitEnabledCurrenciesUpdate` action. Though about doing it on the component, but this will end up being implemented twice with the new onboarding setup, where we can also add new currencies. And also to do it on the PHP side, but the method used to update the currencies is also called internally and could produce _falsy_ events. So I've chosen to do it on the action.

We only need to implement the new event `wcpay_multi_currency_enabled_currencies_updated`, because we already have `wcadmin_inbox_action_click` to track inbox note click. And timestamps are registered on every event, so no need to add it either.

Both events are tracked correctly and connected on funnels, you can take a look at funnel `Multi Currency Inbox note completion` for an example of previous implementation.

#### Testing instructions
**You'll need to wait 5 minutes until live events are available.**
- Click on the Multi-Currency inbox note and look at `Tracks Live View` to check that your event `wcadmin_inbox_action_click` is registered.
- Update enabled currencies adding and removing some of them, and look at `Tracks Live View` to check that your event `wcadmin_wcpay_multi_currency_enabled_currencies_updated` is registered with `added_currencies` and `removed_currencies` in `eventprops`.

#### TODO (After merge)
- [ ] Add the new event to `Tracks in WooCommerce Payments` Field Guide, after merge.

#### Screenshots
<img width="817" alt="Screenshot 2021-09-02 at 09 26 56" src="https://user-images.githubusercontent.com/7670276/131803617-873fecfb-f7f4-401b-8a0a-e1c93226d6d8.png">
<img width="1052" alt="Screenshot 2021-09-10 at 13 16 14" src="https://user-images.githubusercontent.com/7670276/132860217-e622498a-54b7-42a3-9193-77da1a13fa97.png">

-------------------


**Post merge**
